src/plugins/pluginManager.ts

```diff
@@ -119,6 +121,11 @@ const installPlugin = async (url: string): Promise<Plugin | undefined> => {
 
 const uninstallPlugin = async (_plugin: PluginItem) => {
   plugins[_plugin.id] = undefined;
+  storage.mmkv.getAllKeys().forEach(key => {
+    if (key.startsWith(_plugin.id)) {
+      storage.mmkv.delete(key);
```

Comment:
why do you use mmkv's delete method instead of storage instance's one

Reply:
It doesn't have access to `localStorage`, `sessionStorage`.

Reply:
```js
  clearAll(pluginID: string): void {
    const keysToRemove = this.getAllKeys(pluginID);
    keysToRemove.forEach(key => this.delete(pluginID, key));
  }
  ```
  but you declare this

Reply:
The `storage.getAllKeys` function will only return keys that were set using `storage.set`.
It will not return keys to `localStorage`, `sessionStorage`.
(these can only be retrieved via a separate instance and are read-only, [example](https://github.com/Rider21/lnreader-sources/blob/db_2/plugins/russian/ranobelib.ts)).

Reply:
I think I forgot to change the description of the methods.